multipy/runtime/interpreter/interpreter_impl.cpp

```diff
@@ -419,3 +436,24 @@ torch::deploy::InterpreterImpl*
 newInterpreterImpl(const std::vector<std::string>& extra_python_paths) {
   return new ConcreteInterpreterImpl(extra_python_paths);
 }
+
+extern "C" __attribute__((visibility("default")))
+torch::deploy::InterpreterImpl*
+newInterpreterImplSplit(const std::vector<std::string>& extra_python_paths)
+{
+    ConcreteInterpreterImplConstructorCommon(extra_python_paths);
+
+    int r = PyRun_SimpleString(start);
+    TORCH_INTERNAL_ASSERT(r == 0);
+
+    py::object saveStorage = global_impl("torch._deploy", "_save_storages");
```

Comment:
Is there a reason we're using torch._deploy over multipy.utils._deploy https://github.com/pytorch/multipy/blob/0c6ba76ff129c8a63394be8fb36f8d45d5975422/multipy/runtime/interpreter/interpreter_impl.cpp#L210